scripts/linters/pre_commit_linter.py

```diff
@@ -455,6 +456,21 @@ def _print_errors_stacktrace(errors_stacktrace):
         ' above errors gets fixed')
 
 
+def _get_space_separated_linter_name(linter):
+    """Returns the space separated name of the linter class.
+
+    Args:
+        linter: object(LintCheckManager). The sub-linter object to get the
+            linter class name.
+
+    Returns:
+        str. Space separated name of the linter class.
+    """
+    return re.sub(
+        r'((?<=[a-z])[A-Z]|(?<!\A)[A-Z](?=[a-z]))',
```

Comment:
having a seperate function is nice. The regex looks very complicated, can we do this:

`''.join([' ' + c if c.isupper() else c for c in class_name])` wdyt?

Reply:
As per offline discussion, we are going to leave it as it is because of the following reasons:
1. The suggested code won't work for HTMLLinter.
2. We are planning to remove this function in future PR.